packages/flutter/lib/src/widgets/widget_span.dart

```diff
@@ -90,17 +90,28 @@ class WidgetSpan extends PlaceholderSpan {
   /// Helper function for extracting [WidgetSpan]s in preorder, from the given
   /// [InlineSpan] as a list of widgets.
   ///
-  /// The `textScaleFactor` is the the number of font pixels for each logical
-  /// pixel.
+  /// The `textScaler` is the scaling strategy for scaling the content.
   ///
   /// This function is used by [EditableText] and [RichText] so calling it
   /// directly is rarely necessary.
-  static List<Widget> extractFromInlineSpan(InlineSpan span, double textScaleFactor) {
+  static List<Widget> extractFromInlineSpan(InlineSpan span, TextScaler textScaler) {
     final List<Widget> widgets = <Widget>[];
+    // 14.0 is the default font size to use when none of the ancestor spans
```

Comment:
nit: maybe say that it's the default the engine uses in that case?

Reply:
Could also be made self-documenting by defining a const `_kEngineDefaultFontSize`.